packages/npm/financekit/src/index.tsx

```diff
@@ -21,9 +38,12 @@ const Financekit = NativeModules.Financekit
 
 const AuthorizationStatus = S.Literal('authorized', 'denied', 'notDetermined')
 
+export const uuid = S.UUID.pipe(S.brand('CNDL_UUID'))
```

Comment:
```suggestion
export const S_CNDL_UUID = S.UUID.pipe(S.brand('CNDL_UUID'))
```
Should at least start with an uppercase letter like the other schemas. My suggestion is just to match with Candle repo standards

Reply:
which you haven't merge yet...care to share what they are?